QUARKUS-1094.md

```diff
@@ -0,0 +1,24 @@
+# QUARKUS-1094 - Initial support of the gRPC in the dev ui
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1094
+
+Quarkus has an experimental feature for gRPC in the DevUI. Now it is possible to test gRPC services provided by the server without implementing clients. The gRPC uses HTTP2 which is well suited for microservices inner-communication, but there is no native way for browser client to utilize HTTP/2, thus the feature utilizes Web Socket Bridge for communication.
+
+## Scope of the testing
+- Verify all gRPC services are listed in the DevUI
+- Verify all gRPC services are properly identified by their type
+- Verify Unary, Server Streaming, Client Streaming, Bidirectional Streaming are working properly
```

Comment:
Will you reuse existing apps to check the functionality of Dev UI or you need to implement the services first?

Reply:
New app with required services (uni, bi-directional streaming). There is no currently TC using bi-directional streaming. Later this app can be expanded as a skeleton to build a more complex scenario